central/scannerdefinitions/handler/handler.go

```diff
@@ -417,3 +455,50 @@ func openFromArchive(archiveFile string, fileName string) (*os.File, error) {
 	}
 	return tmpFile, nil
 }
+
+func (h *httpHandler) getUpdaterHelper(key string) *requestedUpdater {
+	h.lock.Lock()
+	defer h.lock.Unlock()
+
+	u, exists := h.updaters[key]
+	if !exists {
+		filePath := filepath.Join(h.onlineVulnDir, key+".zip")
+
+		if key == mappingUpdaterKey {
+			url := strings.Join([]string{v4StorageDomain, mappingFile}, "/")
+			h.updaters[key] = &requestedUpdater{
+				RequestedUpdater: newMappingUpdater(
+					file.New(filePath),
+					client,
+					url,
+					h.interval,
+				),
+			}
+		} else {
+			url := strings.Join([]string{scannerUpdateDomain, key, scannerUpdateURLSuffix}, "/")
+			h.updaters[key] = &requestedUpdater{
+				RequestedUpdater: newUpdater(
+					file.New(filePath),
+					client,
+					url,
+					h.interval,
+				),
+			}
+		}
+
+		u = h.updaters[key]
+	}
+
+	u.lastRequestedTime = time.Now()
+	return u
+}
+
+func (h *httpHandler) getMappingFileHelper(w http.ResponseWriter, r *http.Request, fileName string) {
```

Comment:
I don't think `Helper` is a good practice we should start doing. Maybe just `getMappingFile`